doc/sphinx-guides/source/installation/config.rst

```diff
@@ -118,30 +118,15 @@ If you really don't want to front Glassfish with any proxy (not recommended), yo
 
-Root Dataverse Configuration
-----------------------------
-
-The user who creates a dataverse is given the "Admin" role on that dataverse. The root dataverse is created automatically for you by the installer and the "Admin" is the superuser account ("dataverseAdmin") we used in the :doc:`installation-main` section to confirm that we can log in. These next steps of configuring the root dataverse require the "Admin" role on the root dataverse, but not the much more powerful superuser attribute. In short, users with the "Admin" role are subject to the permission system. A superuser, on the other hand, completely bypasses the permission system. You can give non-superusers the "Admin" role on the root dataverse if you'd like them to configure the root dataverse.
-
 Root Dataverse Permissions
-++++++++++++++++++++++++++
-
-In order for non-superusers to start creating dataverses or datasets, you need click "Edit" then "Permissions" and make choices about which users can add dataverses or datasets within the root dataverse. (There is an API endpoint for this operation as well.) Again, the user who creates a dataverse will be granted the "Admin" role on that dataverse.
-
-Publishing the Root Dataverse
-+++++++++++++++++++++++++++++
+--------------------------
 
-Non-superusers who are not "Admin" on the root dataverse will not be able to to do anything useful until the root dataverse has been published.
+The user who creates a dataverse is given the "Admin" role on that dataverse. The root dataverse is created automatically for you by the installer and the "Admin" is the superuser account ("dataverseAdmin") we used in the :doc:`installation-main` section to confirm that we can log in. These next steps of configuring the root dataverse require the "Admin" role on the root dataverse, but not the much more powerful superuser attribute. In short, users with the "Admin" role are subject to the permission system. A superuser, on the other hand, completely bypasses the permission system. You can give non-superusers the "Admin" role on the root dataverse if you'd like them to configure the root dataverse.
 
-Customizing the Root Dataverse
-++++++++++++++++++++++++++++++
+In order for non-superusers to start creating dataverses or datasets, you need click "Edit" then "Permissions" and make choices about which users can add dataverses or datasets within the root dataverse. (There is an API endpoint for this operation as well.) Again, the user who creates a dataverse will be granted the "Admin" role on that dataverse. Non-superusers who are not "Admin" on the root dataverse will not be able to to do anything useful until the root dataverse has been published.
```

Comment:
I'm fine with how the title of this section is not called "Root Dataverse Permissions" but I'm afraid that it reads as a merciless wall of text, starting out with lots of detail on the plumbing rather than stating clearly why the reader should care about any of this. This section is probably the place where we should help the person installing Dataverse avoid several pitfalls. Here are the pain points an solutions:

- I just installed Dataverse and none of my users can do anything. 
  - Publish the root dataverse and configure permissions in order for non-superusers to create dataverses or datasets. See #4515 for an example of this confusion.
- I've been using Dataverse for a little while and I'm not happy with how things are organized.
  - API endpoints to move datasets and dataverses around are being developed. Meanwhile, before you allow lots of dataverses and datasets to be created, you should about what sort of organizational hierarchy of dataverses you want under the root dataverse and adjust the permissions to achieve that goal.
- It's not obvious to me that metadata fields need to be configured.
  - Give your local metadata experts the "Admin" role on various dataverses and have them configure metadata fields. Consider using templates. Notice that you can benefit from facets if you configure them.

There are probably more pain points but these are the ones that come immediately to mind.